code/modules/fluids/air.dm

```diff
@@ -10,10 +10,13 @@ var/list/ban_from_airborne_fluid = list()
 
 	//max_alpha = 200
 
-	required_to_spread = 15
+	required_to_spread = 5
```

Comment:
Is this change (15 -> 5) for non-ass jam intended?

Reply:
oops